### DISCUSSION TITLE
Welcome to OpenSceneGraph Discussions!

### DISCUSSION THREAD

**Discussion by openscenegraph** [openscenegraph/OpenSceneGraph#1174] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ellerychan** [comment]
Okay, I'll start:  Ellery Chan, Precision Lightworks | [PLW Modelworks](https://plwmodelworks.com).  On-and-off user of OpenSceneGraph and OSGEarth over many years.  I have used OSG to prototype concepts and build several visualization utilities that we have used in-house on different projects.  We create 3D city models for flight simulation, urban planning, etc.  Most recently I've written a couple of tiny import plugins to let us use `osgviewer` to QA some of our 3D scenes before we ship them to customers.

Every time I come back to OSG after two or more years I struggle with the build/install process, and the dated documentation frustrates me.  I get it to work but never feel I've mastered it to a level where I'd feel comfortable contributing to updating the docs.  However, despite my private cursing and muttering each time I revisit OSG, I am very grateful and appreciative of the amazing cross-platform functionality it provides, and of all the work that has gone into it.  Thanks, Robert and the community!
